### PR TITLE
UNI-64 Create Navbar Component

### PR DESCRIPTION
**Pull request by TomekStaszkiewicz**
1. Add proper color to theming
2. Implement and test Navbar
3. Implement and test MainLayout
4. Add @material-ui/icons dependency
5. Add proper baseUrl to tsconfig

### CODE REVIEW

File: src/Providers/StyleProvider.tsx [hunk 1]
```diff
@@ -11,6 +11,9 @@ const appTheme = createMuiTheme({
     palette: {
         primary: {
             main: '#fafafa'
+        },
+        secondary: {
+            main: '#282828'
```
Comment (rafalsz98) [inline src/Providers/StyleProvider.tsx:16]: You can discard this changes, so there won't be merge conflict with #3 
btw primary and secondary are accent colors, not background ;d

File: src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    Toolbar,
+    IconButton
+} from '@material-ui/core';
+import { 
+    Menu as MenuIcon, 
+} from '@material-ui/icons';
+
+const Navbar: React.FC = () => {
+    return (
+      <AppBar position="sticky" color="secondary">
```
Comment (rafalsz98) [inline src/components/Navbar/Navbar.tsx:14]: color as "background.dp04".
However it won't compile until on master branch, but what other choice do we have?

File: src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <Toolbar>
+        <IconButton edge="start" color="primary">
```
Comment (rafalsz98) [inline src/components/Navbar/Navbar.tsx:16]: color as "text.primary" should be ok

File: src/components/Navbar/Navbar.tsx (hunk 2 above)
Comment (TomekStaszkiewicz) [reply src/components/Navbar/Navbar.tsx:14]: I can rebase to your branch xD

Comment (TomekStaszkiewicz) [reply src/components/Navbar/Navbar.tsx:14]: Hmm, not sure about this one - AppBar defined  color props as:
```
  type Color = 'inherit' | 'primary' | 'secondary' | 'default';
```

I am not sure if we can customise it. I will check that though

**Review by rafalsz98 — APPROVED** [review]

**Pull request merged**